flink-runtime/src/test/java/org/apache/flink/runtime/execution/librarycache/BlobLibraryCacheRecoveryITCase.java

```diff
@@ -79,7 +79,9 @@ public void testRecoveryRegisterAndDownload() throws Exception {
 
 			final BlobLibraryCacheManager.ClassLoaderFactory classLoaderFactory = BlobLibraryCacheManager.defaultClassLoaderFactory(
 				FlinkUserCodeClassLoaders.ResolveOrder.CHILD_FIRST,
-				new String[0]);
+				new String[0],
+				false,
+				exception -> {});
```

Comment:
Instead of passing a boolean and the fatal error handler to `defaultClassLoaderFactory()` one could also change it to pass in the `exception handler` instead. That way one would not pass in redundant information `false` and `exception -> {}` but it would suffice to simply pass `exception -> {}`.

Reply:
true, we can just make it nullable